movie/d3movie.cpp

```diff
@@ -596,7 +588,7 @@ int NextPow2(int n) {
 void BlitToMovieBitmap(unsigned char *buf, unsigned int bufw, unsigned int bufh, unsigned int hicolor,
                        bool usePow2Texture, int &texW, int &texH) {
   // get some sizes
-  int drawWidth = hicolor ? (bufw >> 1) : bufw;
```

Comment:
What is hicolor doing ? l. 612/620 still tests for it

Reply:
hicolor is indicator that MVE is 16-bit color (there two types of MVEs - 8bit paletted - Dolby Sound intro, and hicolor/truecolor - other intros and videos).
Regarding to changes - there was unnecessary doubling and halving back size of width in old libmve and movie callbacks.